pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -537,16 +537,20 @@ func (r *ReconcileGitTrack) Reconcile(request reconcile.Request) (reconcile.Resu
 	r.recorder.Eventf(instance, apiv1.EventTypeNormal, "CheckoutSuccessful", "Successfully checked out '%s' at '%s'", instance.Spec.Repository, instance.Spec.Reference)
 
 	// Attempt to parse k8s objects from files
-	objects, errors := objectsFrom(files)
-	if len(errors) > 0 {
-		sOpts.parseError = fmt.Errorf(strings.Join(errors, ",\n"))
+	objects, fileErrors := objectsFrom(files)
+
+	// Update status with the number of objects discovered
+	sOpts.discovered = int64(len(objects))
```

Comment:
Perhaps add both here to save a line?
```suggestion
	sOpts.discovered = int64(len(objects)) + int64(len(fileErrors))
```